src/main/java/com/angelbroking/smartapi/sample/Examples.java

```diff
@@ -273,6 +272,20 @@ public void getCandleData(SmartConnect smartConnect) throws SmartAPIException, I
 		String response = smartConnect.candleData(requestObejct);
 	}
 
+    /** Market Data */
+    public void getMarketData(SmartConnect smartConnect) {
+        // Create the payload object
+        JSONObject payload = new JSONObject();

```

Comment:
Cover all 3 modes in 3 different funtion.

Reply:
okay